### ISSUE TITLE
Audio issue on Keitai Deijuu Telefang (GBC)

### ISSUE THREAD

**Issue by Yave-Yu** [mgba-emu/mgba#2863]
0.10.1
In title screen, when game playing PCM sample, weird audio occurs.

**Comment by endrift** [comment]
Yeah, this is a known issue and I've been trying to improve it over time.

**Comment by Blaziken257** [comment]
This issue was not present in mGBA 0.9.3, so it was introduced in 0,10.0. For Telefang, this applies to the retail Japanese versions, as well as the English fan translation, which has a custom voice sample. This issue applies to Medarot 3 as well, which, like Telefang, was developed by Natsume.

For Medarot 4 and 5, the voice samples were silent in mGBA 0.9.3, but have the same buzzing issue as Telefang and Medarot 3 in 0.10.x.
